javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,49 @@
+console.log("Script loaded");
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector("section.products ul");
+console.log(productsUl);
+
+////
+const input = document.getElementById("searchBar");
+input.addEventListener("keyup", () => {
+  let input = document.getElementById("searchBar").value.toLowerCase();
+  const filteredValue = products.filter((product) => {
+    return product.name.toLowerCase().includes(input);
+  });
+  productsUl.innerHTML = " ";
+  renderProducts(filteredValue);
+});
+/////
+renderProducts(products);
+const input1 = document.getElementById("max-price");
+input1.addEventListener("keyup", () => {
+  let maxValue = document.getElementById("max-price").value;
+  const filteredPrice = products.filter((product) => {
+    return product.price < maxValue;
+  });
+  productsUl.innerHTML = " ";
+  renderProducts(filteredPrice);
+});
```

Comment:
Search option is working separately, I mean either it searches by title or max-price. It confuses the user when user enter both and expect both search results.
ex: if user enters both inputs,  I mean "fish" in search-bar and 1000 in max-price, user will expect all fish items of max-price 1000. Now what's happening is, its showing all the items of max-price 1000, its also including other items than fish.

Suggestion:
Check whether user has entered only title or max-price or both, check all the inputs and check for all conditions in filter function. 
Also validate whether user has entered input or not, take only those inputs for search which user has entered.